package.json

```diff
@@ -1,16 +1,21 @@
 {
   "name": "@yext/chat-ui-react",
-  "version": "0.8.2",
+  "version": "0.8.3",
   "description": "A library of React Components for powering Yext Chat integrations.",
   "author": "clippy@yext.com",
   "main": "./lib/commonjs/src/index.js",
   "module": "./lib/esm/src/index.mjs",
-  "types": "./lib/esm/src/index.d.ts",
   "sideEffects": false,
   "exports": {
     ".": {
-      "import": "./lib/esm/src/index.mjs",
-      "require": "./lib/commonjs/src/index.js"
+      "import": {
+        "default": "./lib/esm/src/index.mjs",
+        "types": "./lib/esm/index.d.ts"
+      },
+      "require": {
+        "default": "./lib/commonjs/src/index.js",
+        "types": "./lib/commonjs/src/index.d.ts"
+      }
     },
     "./bundle.css": "./lib/bundle.css"
```

Comment:
can you double check if this still works? It said "resolution failed" in this link: https://arethetypeswrong.github.io/?p=%40yext%2Fchat-ui-react%400.8.3-beta.1 but I'm not sure why that would be. It definitely work based on scrip tag and preview component in alpha

Reply:
The "resolution failed" for the .css is a false positive, it doesn't know how to look for css files.